packages/sdk/src/react/ui/modals/MakeOfferModal/index.tsx

```diff
@@ -124,7 +125,10 @@ const ModalContent = observer(
 			contractType: collection!.type as ContractType,
 			offer: {
 				tokenId: collectibleId,
-				quantity: makeOfferModal$.quantity.get(),
+				quantity: parseUnits(
+					makeOfferModal$.quantity.get(),
+					collectible?.decimals || 0
```

Comment:
Do you think we should inform user that their input with decimals will be considered and order will be created without decimals?
I thought I made offer paying 5 usdc for 4.56 tokens, the request is sent 5 usdc for 4 tokens, to backend. Maybe disabling inputting `.xxx` if the decimals of quantity is 0?